interfaces/offscreen_effect_player.hpp

```diff
@@ -48,6 +48,20 @@ namespace bnb::oep::interfaces
          */
         virtual void process_image_async(pixel_buffer_sptr image, rotation input_rotation, oep_image_process_cb callback, std::optional<rotation> target_orientation) = 0;
```

Comment:
Please do not add a new method in this case but update an existing one. Update at least oep-desktop sample with new module.